tests/phpunit/Integration/JSONScript/TestCases/p-0918.json

```diff
@@ -1,90 +1,24 @@
 {
-	"description": "Test `_eid` with multiple substitutes",
+	"description": "Test #ask with pipe trick output.",
 	"setup": [
```

Comment:
I'm not sure what happened here but the number (or better test) `p-0918.json` is already been occupied, so you should use a number that is free.

Beside the colliding test number, please check that the JSON is valid because currently it says:

```
1) SMW\Tests\Integration\JSONScript\JsonTestCaseScriptRunnerTest::testCaseFile with data set "p-0918.json" ('/home/travis/build/SemanticMe...8.json')
RuntimeException: Expected a JSON compatible format but failed with 'Syntax error, malformed JSON'
```

Reply:
The markdown listed 917 as the last test, so I just blindly put it into 918. Fixed with a real new file, and fixed the JSON and ran all the tests.